client/src/components/application/Find.vue

```diff
@@ -22,13 +22,23 @@
       visble: true
     },
     computed: {
+      auth() {
+        return this.$store.getters.isAuthenticated
+      },
       details() {
         return this.$store.getters.details;
       }
     },
     components: {
       findfilter
     },
+    watch: {
+      auth: {
+        handler(selection) {
+          if (auth != true) { this.$router.push("/Signin") }
```

Comment:
Is this supposed to push me to the login screen when I'm not logged in? It doesn't. I just get the popup (modal) re. not being logged in and the new bold message re. "your authorization is missing or expired...".